config-bigclown.yml

```diff
@@ -7,14 +7,32 @@ influxdb:
   port: 8086
   database: node
 
+http:
```

Comment:
Revert config-bigclown.yml please and create new config example with this changes.
We use config-bigclown.yml in the Grafana  in our documentation.
Thx.